mappings/net/minecraft/entity/mob/BreezeEntity.mapping

```diff
@@ -1 +1,8 @@
 CLASS net/minecraft/unmapped/C_nqresjkz net/minecraft/entity/mob/BreezeEntity
+	FIELD f_gbgtzaaz inhalingState Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_klrmgrpo shootingState Lnet/minecraft/unmapped/C_kxntavoz;
+	FIELD f_ryffbosw longJumpingParticleAddCount I
```

Comment:
```suggestion
	FIELD f_ryffbosw longJumpingTicks I
```
?
It's incremented while `LONG_JUMPING` and reset while `STANDING`
So more precisely it would be `ticksLongJumpingSinceLastTimeStanding`, but I think my suggestion is gets the important part across.
(mojmap is `jumpTrailStartedTick`, yarn is `longJumpingParticleAddCount`, I think both are wrong)